.github/workflows/deploy.yml

```diff
@@ -0,0 +1,66 @@
+name: Hugo Staging Deploy
+
+on:
+  pull_request:
+    types: [review_requested]
+
+jobs:
+  deploy_staging:
+    runs-on: ubuntu-22.04
+    environment: sshdeploy
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    if: ${{ github.event.requested_team.name == 'hackers'}}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive  # Fetch Hugo themes (true OR recursive)
+          fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
+          lfs: true         # Fetch Git LFS files
+
+      # no idea why, but checkout submodules doesn't work properly
+      #- name: git submodule update --init --recursive
+      #  run: git submodule update --init --recursive
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v3
+        with:
+          hugo-version: '0.97.0'
+          extended: true
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+          # The action defaults to search for the dependency file (package-lock.json,
+          # npm-shrinkwrap.json or yarn.lock) in the repository root, and uses its
+          # hash as a part of the cache key.
+          # https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data
+          cache-dependency-path: '**/package-lock.json'
+
+      - name: Install dependencies (postcss), bootstrap-sass
+        run: npm install --save-dev autoprefixer && npm install postcss-cli && npm install -D postcss && npm install -D postcss-preset-env && npm install -D bootstrap-sass
+
+      - name: Build
+        run: cd kernelci.org;hugo --minify
```

Comment:
Don't forget the flags for staging, I guess this could be disabled when the action is running from the `kernelci.org` production branch:

```
-D -F -b https://static.staging.kernelci.org/
```